src/benchmark/file_io_read_micro_benchmark.cpp

```diff
@@ -51,15 +52,22 @@ class FileIOMicroReadBenchmarkFixture : public MicroBenchmarkBasicFixture {
   }
 
  protected:
+  uint32_t allign_to_pagesize(uint32_t buffer_size_mb, uint32_t page_size = 4096);
```

Comment:
Where does `4096` come from? What is it?

Reply:
Typ --> "align"